Lesson 11/src/index.js

```diff
@@ -0,0 +1,43 @@
+// string length
+
+const string10 = 'agdjgfkshv'
+
+console.log(string10.length)
+
+// string reverce
+
+function reverseStr(string) {
+  return string.split('').reverse().join('')
+}
+
+console.log(reverseStr(string10))
+
+// substr
+
+function findSubStr(string, substring) {
+  if (string.indexOf(substring) !== -1) {
+    console.log(true)
+  } else {
+    console.log(false)
+  }
+}
+
+findSubStr('мама мыла раму', 'ла')
```

Comment:
```suggestion
function containsSubstring(string, substring) {
  if (string.indexOf(substring) !== -1) {
    console.log(true)
  } else {
    console.log(false)
  }
}

containsSubstring('мама мыла раму', 'ла')
```

это название лучше подходит, "строка содержит" - логичнее для ответа "да" или "нет", особенно в условиях if и т.д.